typings/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "target": "esnext",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
```

Comment:
JSON isn't allowed to have comments.

Reply:
comments are allowed in tsconfig.json: https://github.com/Microsoft/TypeScript/pull/5450

Reply:
Somebody needs to tell github ;)